sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -471,9 +469,7 @@ export class EventHubSender extends LinkEntity {
           );
 
           try {
-            const senderOptions = this._createSenderOptions(
-              getRetryAttemptTimeoutInMs(options.retryOptions)
-            );
+            const senderOptions = this._createSenderOptions(retryOptions.timeoutInMs!);
```

Comment:
Since `this._sender!.sendTimeoutInSeconds` gets updated later, is there any point in passing the timeout here?

Reply:
Valid point, I can remove that.

Reply:
Okay, I've realized that I can't remove it since the timeout is a required argument, I can change this to default or some other random value.
Any picks?

Reply:
That's ok, we can let it be I guess